editor/src/components/inspector/common/css-utils.ts

```diff
@@ -841,9 +882,27 @@ export const parseCSSUnitlessAsNumber = (input: unknown): Either<string, number>
   }
 }
 
+const validGridDimensionKeywords: string[] = [
```

Comment:
You could make this a type, and then it would be possible to make the code typesafe

Reply:
done in [`d6ffcc7` (#6157)](https://github.com/concrete-utopia/utopia/pull/6157/commits/d6ffcc73f4a76200d91a3efe3769d0404a8461dd)